src/views/EpicBoardView.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { LoginRequired } from "../Auth/Login/helpers/LoginRequired";
+import { Board } from "../Items/Board/Board";
+import { ItemDrawer } from "../Items/Drawer/ItemDrawer";
+import { ItemDrawerProvider } from "../Items/state/ItemDrawerContext";
+import { ItemType } from "../Items/types";
+
+const defaultItem = {
+  type: ItemType.EPIC,
+};
+
+export function EpicBoardView() {
+  return (
+    <LoginRequired>
+      <ItemDrawerProvider defaultItem={defaultItem}>
+        <Board filter={(item) => item.type === ItemType.EPIC} />
```

Comment:
for future - prevent closing epic if some of the epics task are not closed

Reply:
I can check that on the backend though

Reply:
I was thinking about automated status calculation on the basis of child statuses (lowest status of the children)